### ISSUE TITLE
Inquiry about the Mutual Gaze Datasets 

### ISSUE THREAD

**Issue by YimingCuiCuiCui** [Gmbition/MGTR#1]
Thank you for sharing the wonderful work. 

Is there any estimation of when the Mutual Gaze Datasets will be released?

Thank you!

**Comment by csguoh** [comment]
Thank you for your interest in our work. 
We have added download links of the mutual gaze datasets. You can find the link in the README.

**Comment by YimingCuiCuiCui** [comment]
Thank you!

**Comment by YimingCuiCuiCui** [comment]
Hello, can I ask when will the training codes be available? Thank you!

**Comment by csguoh** [comment]
I am sorry that we are not going to publish the training code. However, you can reproduce our work following the steps below. First, our code framework is similar to [DETR](https://github.com/facebookresearch/detr) and [HoiTransformer](https://github.com/bbepoch/HoiTransformer), and you can follow these code for reproducibility. Also, the loss function for training can be found in [here](https://github.com/Gmbition/MGTR/blob/main/models/mgtr.py). Feel free to contact me if you have any questions in the reproduction process.

**Comment by YimingCuiCuiCui** [comment]
Thank you!
